README.md

```diff
@@ -333,6 +333,38 @@ $results = array(/* ... */);
 $adapter = new FixedAdapter($nbResults, $results);
 ```
 
+### AdaptersAdapter
+
+This adapter can take an array of any other adapters and go through them
```

Comment:
Reading the code, this is equivalent to paginating an AppendIterator containing the different iterators being wrapped in adapters